.changeset/nasty-geckos-hide.md

```diff
@@ -0,0 +1,7 @@
+---
+"@aws-amplify/ui-react-core-notifications": patch
+"@aws-amplify/ui-react-core": patch
+"@aws-amplify/ui-react": patch
+---
```

Comment:
Should we have a changeset all here? Shouldn't the primary PR against `main` contain one that covers everything?

Reply:
Oh yeah... that's fair. I'm good to move the changeset content to the primary one.

Reply:
okay, will remove the one here, and add in the [main PR](https://github.com/aws-amplify/amplify-ui/pull/3824) with a message explaining all the changes.